### PR TITLE
Moving to dronecode sdk

### PR DESCRIPTION
**Pull request by xvzf**
@julianoes should be it!

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -11,13 +11,13 @@ Clone the repo (or your fork) and recursively update submodules:
...
 
-    git clone https://github.com/dronecore/DroneCore-Python.git --recursive
+    git clone https://github.com/dronecore/DronecodeSDK-Python.git --recursive
 
 This can be done as 2 steps, as shown below.
 
 .. code-block:: bash
 
-    git clone https://github.com/<your fork>/DroneCore-Python.git
+    git clone https://github.com/<your fork>/DronecodeSDK-Python.git
```
Comment (julianoes) [inline README.rst]: I would probably remove the `<your fork>`.

Comment (hamishwillee) [reply README.rst]: I like the `<your fork>` but it should be applied consistently - ie in the previous single-step line too.

Comment (julianoes) [reply README.rst]: But we are not actually trying to promote forks...

Comment (hamishwillee) [reply README.rst]: OK. I tend to assume that the model for contribution is that you branch on your fork and then submit a PR there. If not, then fair enough.

Comment (julianoes) [reply README.rst]: Hm, I see, for development. You're right, and that's what this is for.

Comment (julianoes) [reply README.rst]: I was thinking more about this and found that actually, we should not have to explain these things. Developers either know this or should google "how to make a github pull request".

Comment (hamishwillee) [reply README.rst]: I don't feel strongly about that, but I do about the inconsistency. More important, the fact that the repo moved was bugging me. I have fixed it how @julianoes would like it in https://github.com/Dronecode/DronecodeSDK-Python/commit/d20e1be3b26594a81668df5e769a727cc1a1d293

Comment (julianoes) [reply README.rst]: Thx.

**Pull request merged**